.github/workflows/CI.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           - os: ubuntu-latest
             python-version: "3.11"
             mpi: ""
-            doctest: true
+            doctest: false
```

Comment:
no doctest anymore?

Reply:
check the line above, we were running it on both python 3.10 and 3.11. This way we run standard tests on python 3.11 (and we keep the doctests on python 3.10).

I think it was a typo from a past PR.